fuzzing/src/visitors/facevisitor-colrv1.cpp

```diff
@@ -0,0 +1,245 @@
+// facevisitor-colrv1.cpp
+//
+//   Finds and traverses COLRv1 glyphs using FreeType's
+//   COLRv1 API.
+//
+// Copyright 2021 by
+// Dominik Röttsches.
+//
+// This file is part of the FreeType project, and may only be used,
+// modified, and distributed under the terms of the FreeType project
+// license, LICENSE.TXT.  By continuing to use, modify, or distribute
+// this file you indicate that you have read the license and
+// understand and accept it fully.
+
+
+#include "visitors/facevisitor-colrv1.h"
+
+#include <cassert>
+
+#include "utils/logging.h"
+
+#include <ft2build.h>
+#include FT_FREETYPE_H
+
+namespace {
+
+bool colrv1_start_glyph( const FT_Face& ft_face,
+                         uint16_t glyph_id,
+                         FT_Color_Root_Transform root_transform );
```

Comment:
this applies to all function signatures: the way that FreeType uses that in the C codebase (and what I tried to mimic) is the following layout (e.g.):

```cpp
bool colrv1_start_glyph( const FT_Face&           ft_face,
                         uint16_t                 glyph_id,
                         FT_Color_Root_Transform  root_transform );
```

- let's left-align all variable names
- let's have two spaces between the type + the name

Reply:
Done, hope I didn't miss any occasions.